discord/cache.py

```diff
@@ -1,122 +1,46 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, OrderedDict
+from typing import TYPE_CHECKING, Any, Dict, OrderedDict
+
+from .types.snowflake import Snowflake
+
 
 if TYPE_CHECKING:
 
     from .api.httphandler import HTTPHandler
     from .guild import Guild
     from .message import Message
     from .role import Role
-    from .types.snowflake import Snowflake
     from .user.member import Member
     from .user.user import User
 
 
-class LFUCache:
-
-    capacity: int
```

Comment:
Why are we removing the properties?

Reply:
Only the typehint is removed, the `capacity` is still there

Reply:
Imo the typehint should still exist.
Cause it helps with code readablility.

Reply:
it is still there?

Reply:
The typehint is still there but now it is in the `__init__`